Clinical-Genomics/scout
# Pre-select the show all vars checkbox using Institute settings

Fix #3541 

- Improve the looks of Institute Settings page
- Adds a new settings option in institutes settings that will force variants filter form to return all variants (not only those present in affected individuals

<img width="400" alt="image" src="https://user-images.githubusercontent.com/28093618/186166238-75714cac-08a8-4a14-b929-870dfb653cd8.png">

This will cause the variants filter to return all variants:

<img width="333" alt="image" src="https://user-images.githubusercontent.com/28093618/186166503-cb49cfa7-ef9d-45d6-a014-2e529ecba7b5.png">


<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the paxa software installed on `hasta`:
 - Log in into hasta: `ssh <USER.NAME>@hasta.scilifelab.se`
 - Activate the staging environment with the command `us` (Use Stage)
 - Run the command `paxa` and follow the instructions. Make sure to specify scout-stage as the resource you request and the server cg-vm1 as server.
1. Log out from the hasta server.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
1. After testing is done, log out from `cg-vm1` and log in again in the `hasta` server, repeat the `hasta` and `paxa` procedure, which will release the allocated resource (scout-stage) to be used for testing by other users.
</details>


**How to test**:
1. Install on cg-vm1 
2. Edit the settings of one institute to pre-select the `Preselected 'Show also variants only present in unaffected'` checkbox
3. Check variants page (from variants of different type) to make sure that all vars are returned and not only those from affected individuals

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
